### PR TITLE
Make peekedController a val

### PR DESCRIPTION
**Pull request by shadaj**

### CODE REVIEW

File: core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala [hunk 1]
```diff
@@ -54,6 +58,10 @@ abstract class Component[T](period: Time)(implicit val clock: Clock) {
     }
     // scalastyle:on
```
Comment (PhilipAxelrod) [inline core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala:59]: I think you forgot to remove this

File: core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala [hunk 2]
```diff
@@ -54,6 +58,10 @@ abstract class Component[T](period: Time)(implicit val clock: Clock) {
...
 
+    val value = currentController.currentValue(dt)
+
+    lastOutput = Some(value)
```
Comment (PhilipAxelrod) [inline core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala]: is "value" public by intention? I'd just do 
Some(currentController.currentValue(dt))

**Review by PhilipAxelrod — CHANGES_REQUESTED** [review]

File: core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala (hunk 1 above)
Comment (shadaj) [reply core/shared/src/main/scala/com/lynbrookrobotics/potassium/Component.scala:59]: No, we want to keep it, because otherwise we have style complaints.

**Review by PhilipAxelrod — APPROVED** [review]
lgtm

**Comment by PhilipAxelrod** [comment]
Also, you need to rebase I think

**Pull request merged**